relays/substrate/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "substrate-bridge"
+version = "0.1.0"
+authors = ["Jim Posen <jim.posen@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+async-std = "1.0.1"
+clap = "2.3.3"
+ctrlc = "3.1.3"
+derive_more = "0.99.1"
+env_logger = "0.7.1"
+futures = "0.3.1"
+jsonrpsee = { git = "https://github.com/paritytech/jsonrpsee", features = ["ws"], branch = "jimpo/positional-params" }
```

Comment:
Can we merge that changes upstream first? I don't want to depend on possibly temporary branches.

Reply:
I've made a PR (paritytech/jsonrpsee#94) for this.